sdk/vc-data/package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@affinidi/vc-data",
-  "version": "1.3.2",
+  "version": "1.3.3",
```

Comment:
You should also update version on line 9

Reply:
ran `npm install` it fixed this and also added some missing dependencies.

Reply:
per inga's comment on Slack(should only use lerna bootstrap in monorepo), I reverted `npm install` changes and manually changed the line 9.